kernel/kernel-api/src/main/java/io/delta/kernel/TableVersionNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package io.delta.kernel;
+
+/**
+ * Thrown when a particular version is not found in the Delta table
+ */
+public class TableVersionNotFoundException
```

Comment:
Can we remove this? Seems unused

Reply:
yeah, I think we can remove it now and add once we support getting a snapshot for a specific version.